src/org/opensolaris/opengrok/configuration/RuntimeEnvironment.java

```diff
@@ -98,6 +98,7 @@
 import static java.nio.file.StandardWatchEventKinds.ENTRY_CREATE;
 import static java.nio.file.StandardWatchEventKinds.ENTRY_DELETE;
 import static java.nio.file.StandardWatchEventKinds.ENTRY_MODIFY;
+import java.util.HashSet;
```

Comment:
did nb order the imports in this manner? I thought always that static imports must follow normal imports